### PR TITLE
[Add] Nav Menu

### PR DESCRIPTION
**Pull request by PTaylour**
## ⚠️ I don't have push access yet so this is from a fork

# yooo

## This PR

- adds responsive Nav
- on small screens shows hamburger and drawer menu
- on large screens shows a normal list of links
- also adds ESLint and Chakra

image

image

image

## In future PRS

- update links to point at the right pages

### CODE REVIEW

File: src/components/layout.js [hunk 1]
```diff
@@ -11,7 +11,6 @@ import { useStaticQuery, graphql } from "gatsby"
 
 import Header from "./header"
 import Footer from "./footer"
-import "./layout.css"
```
Comment (keithbro) [inline src/components/layout.js:14]: 👏 👏 👏

File: src/components/header.js [hunk 2]
```diff
@@ -1,31 +1,100 @@
...
+          display={["none", "none", "block"]}
+          as="nav"
+          direction="row"
+          spacing={2}
+        >
+          {PAGES.map(page => (
+            <NavLink key={page.children + page.to} {...page} />
+          ))}
+        </Stack>
+        <Drawer isOpen={isOpen} placement="right" onClose={onClose}>
```
Comment (keithbro) [inline src/components/header.js:70]: Nicely done.

File: package.json [hunk 3]
```diff
@@ -9,9 +9,13 @@
     "@emotion/core": "^10.0.28",
     "@emotion/styled": "^10.0.27",
     "emotion-theming": "^10.0.27",
+    "eslint": "^6.8.0",
```
Comment (keithbro) [inline package.json:12]: Might be better to move these to devDeps - but we can do so in a future PR

**Review by keithbro — APPROVED** [review]

File: package.json (hunk 3 above)
Comment (PTaylour) [reply package.json:12]: Yeah true say. Although `gatsby-plugin-eslint` is used at build time so it may be necessary in CI?

Comment (PTaylour) [reply package.json:12]: thanks for the review btw :)

**Pull request merged**